src/structurizr/workspace.py

```diff
@@ -195,18 +195,54 @@ def __init__(
 
     @classmethod
     def load(cls, filename: Union[str, Path]) -> "Workspace":
-        """"""
+        """Load a workspace from a file, which may be zipped."""
         filename = Path(filename)
         try:
             with gzip.open(filename, "rt") as handle:
-                ws_io = WorkspaceIO.parse_raw(handle.read())
+                return cls.loads(handle.read())
         except FileNotFoundError as error:
             raise error
         except OSError:
             with filename.open() as handle:
-                ws_io = WorkspaceIO.parse_raw(handle.read())
+                return cls.loads(handle.read())
+
+    @classmethod
+    def loads(cls, json_string: str) -> "Workspace":
+        """Load a workspace from a JSON string."""
+        ws_io = WorkspaceIO.parse_raw(json_string)
         return cls.hydrate(ws_io)
 
+    def dump(
+        self,
+        filename: Union[str, Path],
+        *,
+        zip: bool = False,
+        indent: Optional[int] = None,
```

Comment:
Since all keyword arguments are passed through `dumps` to the `json` method eventually, what is your reason to make `indent` more visible here than other arguments?

Reply:
I thought it worth calling out as it controls whether to pretty-print or not which someone not familiar with `json.dumps` would likely have to search for.  Happy to remove if you think it's not worth it.

Reply:
Nah, happy to keep in.